app/playersinscription/addPlayerBusiness.js

```diff
@@ -0,0 +1,63 @@
+import * as common from '../../utils/common';
```

Comment:
This module says business, but it mixes validation and UI (HTML). Try to separate concerns cleanly.